subjects/mini-framework/README.md

```diff
@@ -0,0 +1,112 @@
+## mini-framework
+
+Now that you have already used a framework of your choice, you must now implement some features on a framework of your own. That's right, you are going to create a framework.
+
+Be aware that a framework is different from a library. When you call a method from a library, you are in control. But with a framework, the control is inverted: the framework calls you.
+
+### Objectives
+
+Your framework should implement:
+
+- Abstracting the DOM
+  Routing System
+- State Management
+- Event Handling
+
+You will also need to make a [todoMVC](http://todomvc.com/) project using your framework.
+
+### Instructions
+
+You must create documentation for your framework, so that users (auditers) are able to understand and know how to use your framework without experiencing any awkwardness. By documentation we mean, the explaining of how does the framework works and how to work with it, for example: how to create a div, how to add an event to a button, etc.
+
+Your framework will be tested by using it, like you previously have used one, in the social network project. So the user has to be presented to a folder structure that allows him to run the app from the root of that folder structure. The user testing your framework will have to implement some simple code in order to test the features described bellow.
```

Comment:
I think documentation should have it's own big title so it's not overlooked by students.
We should make a few points like:
Must be in markdown
Include code example
a complete reference that cover all the framework API
explain the reason why things are done like this.